core/v0/api/core.yaml

```diff
@@ -1263,9 +1263,7 @@ components:
         bpp/providers:
           type: array
           items:
-            allOf:
-              - $ref: '#/components/schemas/Provider'
-              - $ref: '#/components/schemas/ProviderCatalog'
+            $ref: '#/components/schemas/Provider'
```

Comment:
Same comments as in the json version

Reply:
Discussed on the call. Decided apply this change.